### ISSUE TITLE
Plugin custom media categories beserk?

### ISSUE THREAD

**Issue by rica-carv** [e107inc/e107#2732]
I'm trying to use custom media categories on my plugin, but i've stumble in two "issues":

1. When installing the plugin, it setups the media categories for use under media manager, but it goes and puts the images already on the images folder of the plugin under the plugin media category! Is there any way i can tell that those images are not to put under the plugin custom media category? But of course, they can be put on the "global" images media category....

2. How can i limit the images picked from media manager to the plugin media category only, at start?
It's kind of ankwards to have a plugin media category for images to be used on the plugin itself, and then have to be scavenging thorught all the images when picking one on the plugin management area....

**Comment by Jimmi08** [comment]
@rica-carv look at banner plugin. You can set media category for image field in admin_ui

**Comment by rica-carv** [comment]
@Jimmi08 Yes, i know, but i want to make it "hardlinked" to the plugin, and therefore, i want it to be automatically created on plugin install....

**Comment by Moc** [comment]
@CaMer0n Your thoughts please :)

**Comment by CaMer0n** [comment]
@rica-carv Do you have something like this in your plugin.xml ?

```
	<mediaCategories>
		<category type="image">Banners</category>
	</mediaCategories>
```

**Comment by rica-carv** [comment]
@CaMer0n Yes, i do....
